README.md

```diff
@@ -101,3 +97,24 @@ Then run the application (here on Android):
 ```sh
 npm run cordova-android
 ```
+
+## How to build the Chrome extension ``hoverpad.crx`` file?
+
+Assuming the ``chromium-browser`` command is available in your ``PATH``:
```

Comment:
So basically the idea is to make it available in you path.

For instance:
```js
ln -s /opt/google/chrome/chrome /usr/local/bin/chromium-browser
```

Or
```
ln -s /Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome /usr/local/bin/chromium-browser
```